client/src/pages/choosePlayers.tsx

```diff
@@ -34,17 +45,17 @@ export default function ChoosePlayers() {
 
   const selectedEqual =
     instruments.filter((instrument) => instrument.selected).length ===
-    names.filter((name) => name.selected).length;
+    musicians.filter((name) => name.selected).length;
 
-  const handleClickName = (item: Name) => {
-    const nextName = names.map((name) => {
+  const handleClickMusician = (item: Name) => {
+    const nextName = musicians.map((name) => {
```

Comment:
could the name value you are passing here be "musician" instead? I think it would help make the next line a bit easier to read 'musician.name' instead of 'name.name' --- what do you think?